sonar-deprecated/src/main/java/org/sonar/api/resources/Library.java

```diff
@@ -21,6 +21,10 @@
 
 import org.apache.commons.lang.builder.ToStringBuilder;
 
+/**
+ * @deprecated since 5.2 No more design features
+ */
+@Deprecated
 public final class Library extends Resource {
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)